Source/WebCore/css/parser/CSSPropertyParserConsumer+Color.cpp

```diff
@@ -777,7 +777,7 @@ static std::optional<SRGBA<uint8_t>> consumeHexColor(CSSParserTokenRange& range,
             else
                 string = makeString(integerValue, token.unitString()); // e.g. 0001FF
             if (string.length() < 6)
-                string = makeString(&"000000"[string.length()], string);
```

Comment:
Is there a way to disallow this in `makeString` ?

Reply:
Yes, it will be straightforward to do that. One way to do it is to overload `makeString` and use `= delete`.

Reply:
Disallow what?